release/kubernetes-manifests.yaml

```diff
@@ -265,6 +265,8 @@ spec:
           #   value: "1"
           # - name: JAEGER_SERVICE_ADDR
           #   value: "jaeger-collector:14268"
+          # - name: CYMBAL_LOGO
```

Comment:
You only need to modify `kubernetes-manifests/` -- not `release/kubernetes-manifests`. 
When we do the release, any YAML changes will land in `release/`. But we don't manually edit `release/` files. 

You can check out the release scripts for a preview! https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/hack

Reply:
Done (`release/kubernetes-manifests` is now untouched)! Thanks, Megan!